lib/utils.js

```diff
@@ -441,25 +441,35 @@ const executeOperation = (topology, operation, args, options) => {
   });
 };
 
+/**
+ * Applies retryWrites: true to a command if retryWrites is set on the command's database.
+ *
+ * @param {object} target The target command to which we will apply retryWrites.
+ * @param {object} db The database from which we can inherit a retryWrites value.
+ */
+function applyRetryableWrites(target, db) {
```

Comment:
This preserves our current behavior, so we don't need to decide this now. However, I wonder if we are expected to support `retryWrites` at the collection/operation level.

Reply:
According to the [spec](https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst), "This option MUST NOT be configurable at the level of a database object, collection object, or at the level of an individual write operation."

Reply:
ok, awesome!